plugins/dev-tools/README.md

```diff
@@ -47,5 +47,24 @@ addGUIControls((options) => {
 }, defaultOptions);
 ```
 
+### Populate Random
+
+The `populateRandom` function adds random blocks to a workspace. Blocks are selected from the full set of defined blocks. Pass in a worskpace and how many blocks should be created.
```

Comment:
The playground might also want to have a button that does this by default. Although not against still exporting it (similar to what we do with the DebugRenderer)